include/fi_enosys.h

```diff
@@ -170,23 +170,12 @@ int fi_no_reject(struct fid_pep *pep, fi_connreq_t connreq,
 int fi_no_shutdown(struct fid_ep *ep, uint64_t flags);
 
 /*
-static struct fi_ops_av X = {
-	.size = sizeof(struct fi_ops_av),
-	.insert = X,
-	.insertsvc = X,
-	.insertsym = X,
-	.remove = X,
-	.lookup = X,
-	.straddr = X,
-};
-*/
```

Comment:
I just repositioned the fi_ops_av and filled it in with the fi_no_* that were available. If those weren't supposed to be there then I can update the pull request. I'm not exactly sure what is required/not required so I just went with what was already defined.